packages/ubuntu_desktop_installer/lib/pages/connect_to_internet/wifi_model.dart

```diff
@@ -0,0 +1,314 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:collection/collection.dart';
+import 'package:dbus/dbus.dart';
+import 'package:flutter/material.dart';
+
+import '../../services.dart';
+import 'connect_model.dart';
+import 'network_device.dart';
+import 'property_stream_notifier.dart';
+
+// TODO: appropriate timeout?
```

Comment:
If it's any reference, gnome-control-center's Wi-Fi panel triggers a periodic scan every 15 seconds, which lasts for 3 seconds.

Reply:
Thanks! I've added the same